test/functional/ex_cmds/encoding_spec.lua

```diff
@@ -25,7 +25,7 @@ describe('&encoding', function()
     clear('--cmd', 'set enc=latin1')
```

Comment:
Yes, but that is not a problem if we currently don't have any option of that kind. :)